gcp-function-pubsub/README.md

```diff
@@ -37,12 +37,14 @@ module "gcp-function-pubsub-trigger" {
 | runtime                        | The runtime in which to run the function.                                                                                                                                       | string       |           | YES      |
 | all_traffic_on_latest_revision | Whether 100% of traffic is routed to the latest revision. Defaults to true.                                                                                                     | bool         | TRUE      | NO       |
 | available_memory               | The amount of memory available for a function. Defaults to 128M. Supported units are k, M, G, Mi, Gi. If no unit is supplied the value is interpreted as bytes.                 | string       | 128Mi     | NO       |
-| environment_variables          | User-provided build-time environment variables for the function.                                                                                                                | map(string)  | {}        | NO       |
+| build_environment_variables    | User-provided build-time environment variables for the function.                                                                                                                | map(string)  | {}        | NO       |
+| environment_variables          | User-provided runtime environment variables for the function.                                                                                                                   | map(string)  | {}        | NO       |
+| secret_environment_variables   | User-provided runtime environment variables sourced form Google Sercret's Mangager.                                                                                             | list(object) | {}        | NO       |
```

Comment:
Why are the secret_environment variables not also map(string) to keep it consistent?

Reply:
Mostly b/c secret environment variables aren't itself simple key value pairs. Also to not by accident to mistake them as regular env variables.